infra/nightly.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# The primary purpose of this script is to run all the tests and upload 
+# the results to the nightly-results server. It also generates some HTML
+# to display the equiderivability tests in a nicely-formatted way.
+
+echo "Beginning Ruler nightly script..."
+
+# exit immediately upon first error
+set -e
+
+# determine physical directory of this script
+src="${BASH_SOURCE[0]}"
+while [ -L "$src" ]; do
+  dir="$(cd -P "$(dirname "$src")" && pwd)"
+  src="$(readlink "$src")"
+  [[ $src != /* ]] && src="$dir/$src"
+done
+MYDIR="$(cd -P "$(dirname "$src")" && pwd)"
+
+# Absolute directory paths
+TOP_DIR="$MYDIR/.."
+RESOURCE_DIR="$MYDIR/nightly"
+NIGHTLY_DIR="$TOP_DIR/nightly"
```

Comment:
Could possibly be a good idea to call these different things, perhaps nightly_resources and nightly_output or something.

Reply:
Yep, probably a good idea to at least rename one of them.